src/main/webapp/style.css

```diff
@@ -0,0 +1,35 @@
+#chat {
+  align-items: center;
+  border-width: 1px;
+  border-style: solid;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 10px 10px;
+
+}
+
+#chatbox {
+  background: #ddd;
+  height: 500px;
+  margin: 10px 0;
+  overflow-y: scroll;
+  padding: 5px 10px;
+  width: 300px;
+}
+
+.message-anouncement {
+  color: #b34700;
+}
+
+.message-correct {
+  color: green;
```

Comment:
stick to using the hex code.

Also, since this is now the "global" css, its probably worth defining these colors as variables to better describe them, and then referring to the variables in these classes.

This will also help the app look consistent when other contributors are styling their parts.

Check out this documentation for how to do this:
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties